spring/src/main/java/com/connexta/transformation/rest/spring/TransformApiTransform.java

```diff
@@ -84,12 +94,19 @@
       produces = {"application/json"},
       consumes = {"application/json"},
       method = RequestMethod.POST)
+  @ResponseHeader(
+      name = "Content-Version",
+      description =
+          "The API version used by the server to produce the REST message. The server will accept messages for any minor versions prior to this one.",
+      response = String.class)
   default ResponseEntity<Void> transform(
```

Comment:
❓ Don't we need to add a 503 as a possible response to this endpoint as well?

Reply:
No because that endpoint doesn't load anything that could be of a newer version. It simply creates it. We are using 503 as our way to tell the client to retry again where they should be hitting a newer version of the endpoint service.